packages/nameguard-js/src/nameguard-js.test.ts

```diff
@@ -34,37 +34,75 @@ describe("NameGuardJS", () => {
     const localNameguard = createClient({
       // not a real endpoint, will error if used
       endpoint: "http://localhost:1234",
```

Comment:
It's best to define this as a named constant such as `INVALID_NAMEGUARD_API_ENDPOINT` and not as a hardcoded string, especially if it is repeated in multiple places.

But more deeply, why are we defining this unit test in a way that we want it to fail? Why not leave the `endpoint` param as the default which will use `api.nameguard.io` and then write this unit test such that it should succeed?

Reply:
Without this we wouldn't be sure if the API was actually used or if nameguard-js "cheated" somehow.

Reply:
With computeNameGuardReport this shouldn't be possible (since nameguard-js cannot generate the report) but I think this invalid endpoint makes that more explicit.